libs/wingc/src/type_check.rs

```diff
@@ -354,14 +365,11 @@ impl Subtype for Type {
 
 				// In this section, we check if the parameter types are not subtypes of each other, then this is not a subtype.
 
-				// Check that this function has at least as many required parameters as the other function requires
-				if l0.min_parameters() < r0.min_parameters() {
-					return false;
-				}
```

Comment:
I removed this for now since it was causing issues with type checking against interfaces... for now I've left type checking for matching # of params as a future TODO. Related: #1678

Reply:
lets not do this change. We'll fix this to something correct in #1678, but I don't want to hurt function comparison for this. See comment above.

Reply:
Ok, added it back